lms/services/blackboard_api/_basic.py

```diff
@@ -70,6 +70,13 @@ def get_token(self, authorization_code):
             authorization_code=authorization_code,
         )
 
+    def refresh_access_token(self):
+        self._oauth_http_service.refresh_access_token(
+            self.token_url,
+            self.redirect_uri,
+            auth=(self.client_id, self.client_secret),
+        )
```

Comment:
This is exactly the same as the existing code for doing a Blackboard refresh:

https://github.com/hypothesis/lms/blob/92eba8703f450a999f3f01ed055588a8c32a578e/lms/services/blackboard_api/_basic.py#L79-L83

This existing code will be removed soon, when the `frontend_refresh` feature flag is removed

Reply:
Why not call the new method on `request()`?

To make the deletion part later on clearer?